includes/db/LoadBalancer.php

```diff
@@ -370,12 +370,13 @@ public function waitFor( $pos ) {
 	/**
 	 * Set the master wait position and wait for ALL slaves to catch up to it
 	 * @param $pos int
+	 * @param $wiki
 	 */
-	public function waitForAll( $pos ) {
+	public function waitForAll( $pos, $wiki ) {
```

Comment:
To be consistent, `$wiki` would be an optional parameter here too defaulting to `false`